### PR TITLE
Fe header

### PR DESCRIPTION
**Pull request by berkbozok**
All header files added into src files for the issue #79

### CODE REVIEW

File: Client/src/App.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
+/** [App.js]
+* Summary.
+AuthProvider wraps the Routes so that we can check when loggedIn is true.
```
Comment (Pyxsys) [inline Client/src/App.js:3]: Uncertain if app.js has a differing purpose in front-end. @MichaelDCristiano can you confirm?

File: Client/src/Components/Inventory/InventorySideBar/AddInventoryForm.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ const AddInventoryForm = (props) => {
     return (
         <div className = "">
             <form className = "" onSubmit = {submitForm}>
-                <h1>Add Inventory Information</h1>
+                <h1>Add Inventory Information TEST4</h1>
```
Comment (Pyxsys) [inline Client/src/Components/Inventory/InventorySideBar/AddInventoryForm.js]: Remove "TEST 4". Deliverables should be free of visible 'dev-code'.

File: Client/src/Components/Inventory/InventorySideBar/InventorySideBar.js [hunk 3]
```diff
@@ -1,3 +1,14 @@
+/** [InventorySlideBar.js]
```
Comment (Pyxsys) [inline Client/src/Components/Inventory/InventorySideBar/InventorySideBar.js]: correct to "Inventory*Side*Bar"

File: Client/src/Components/Inventory/InventorySideBar/Inventory.js [hunk 4]
```diff
@@ -1,13 +1,20 @@
+/** [Inventory.js]
+* Summary.
+The inventory function is using the slidebar and its routes
```
Comment (Pyxsys) [inline Client/src/Components/Inventory/InventorySideBar/Inventory.js]: Correct "slidebar" to *sidebar*

File: Client/src/Components/Inventory/InventorySideBar/Inventory.js [hunk 5]
```diff
@@ -1,13 +1,20 @@
...
+* 
+* Description. 
+Thanks to slidebar and its routes we can receive the information for Inventory and parts needed for the inventory.
```
Comment (Pyxsys) [inline Client/src/Components/Inventory/InventorySideBar/Inventory.js]: Correct "slidebar" to *sidebar*

File: Client/src/Components/Manufacturing/Manufacturing.js [hunk 6]
```diff
@@ -1,3 +1,9 @@
+/** [Manufacturing.js]
+* Summary. (Use periods, 1-2 sentences MAX)
+* 
+* Description. (Use periods)
```
Comment (Pyxsys) [inline Client/src/Components/Manufacturing/Manufacturing.js:4]: Nothing submitted.

File: Client/src/Components/Navbar/NavBarButton.js [hunk 7]
```diff
@@ -1,3 +1,11 @@
+/** [NavBarButton.js]
+* Summary.
+The required functions and visual design of the code is implemented.
+Once clicked it will bring the elements for the menu.
+*
```
Comment (Pyxsys) [inline Client/src/Components/Navbar/NavBarButton.js:4]: swap the contents of the summary and description.

File: Client/src/Forms/FormLogin.js [hunk 8]
```diff
@@ -1,3 +1,12 @@
+/** [FormLogin.js]
+* Summary. 
+The purpose of this page is to show user an Interface where they can login into their account.
+* 
+* Description. 
+By using FrontEnd and BackEnd simultaneously, the user will login his/her account and it will be checked on the data base
```
Comment (Pyxsys) [inline Client/src/Forms/FormLogin.js]: Statement not true. Front-end sends a request login form to Back-end. Back-end will approve or reject the login. steps are done sequentially not simultaneously.

File: Client/src/Forms/FormLogin.js [hunk 9]
```diff
@@ -13,9 +22,10 @@ const FormLogin = (props) => {
         props.submitForm
     )
     
-    return (
-    <div className ="form-content-login">
-          <form className= "form" onSubmit={handleSubmit}>
+    return (  
+    <div className ="form-content-login" >
+       <form className= "form" onSubmit={handleSubmit}>
+          <img src="https://user-images.githubusercontent.com/58408904/110261377-03b91480-7f7e-11eb-894d-87afe991198b.png" alt="logo" width="300" height="300"  />
```
Comment (Pyxsys) [inline Client/src/Forms/FormLogin.js:29]: Explain change here and note any related issues. Reply to this comment to do so.

File: Client/src/Forms/FormSuccess.js [hunk 10]
```diff
@@ -1,3 +1,12 @@
+/** [FormSuccess.js]
+* Summary.
+After user user just finished creating profile, the system will show success message.
```
Comment (Pyxsys) [inline Client/src/Forms/FormSuccess.js]: * repeated word "user".
* modify to "the system will *display a* success message"

File: Client/src/Forms/useForm.js [hunk 11]
```diff
@@ -1,11 +1,17 @@
+/** [useForm.js]
+* Summary. 
+This useForm is for the sign up page, it checks if the values entered in the form are correct, then handles the submit.
```
Comment (Pyxsys) [inline Client/src/Forms/useForm.js]: Swap contents of Summary and Description.

File: Client/src/Forms/validateInfo.js [hunk 12]
```diff
@@ -1,5 +1,11 @@
-
-//Ensuring no invalid emails, username, or passwords are given through the signup form.
+/** [validateInfo.js]
+* Summary. 
+Ensuring no invalid emails, username, or passwords are given through the signup form.
+* 
+* Description.
+Cheking user's input to make sure that they fill user register details as the website requires.
+Each input box has its own rules and we are checking that we get correct/accurate input from user.
```
Comment (Pyxsys) [inline Client/src/Forms/validateInfo.js]: Change "correct/accurate" to *valid*.

File: Client/src/Forms/validateLogin.js [hunk 13]
```diff
@@ -1,6 +1,13 @@
+/** [validateLogin.js]
+* Summary. 
+Validating that the login values are real.
+*
+* Description. 
+Showing user that input boxes needs to be filled in orde to make a successful login.
```
Comment (Pyxsys) [inline Client/src/Forms/validateLogin.js]: Typo: "... to be filled in ***orde*** to make successful login"

**Review by Pyxsys — CHANGES_REQUESTED** [review]
Several files need changes, please refer to provided comments.
Aim to use third-person stative language, like in ENCS 282 (eg. "This form gathers login information from a user."), when resolving the requested changes.

File: Client/src/Components/Manufacturing/Manufacturing.js (hunk 6 above)
Comment (berkbozok) [reply Client/src/Components/Manufacturing/Manufacturing.js:4]: Michael said he is working on it file so it is empty therefore not much to write

File: Client/src/Components/Inventory/InventorySideBar/AddInventoryForm.js (hunk 2 above)
Comment (berkbozok) [reply Client/src/Components/Inventory/InventorySideBar/AddInventoryForm.js]: done

File: Client/src/Components/Inventory/InventorySideBar/InventorySideBar.js (hunk 3 above)
Comment (berkbozok) [reply Client/src/Components/Inventory/InventorySideBar/InventorySideBar.js]: done

File: Client/src/Components/Inventory/InventorySideBar/Inventory.js (hunk 4 above)
Comment (berkbozok) [reply Client/src/Components/Inventory/InventorySideBar/Inventory.js]: done

File: Client/src/Components/Inventory/InventorySideBar/Inventory.js (hunk 5 above)
Comment (berkbozok) [reply Client/src/Components/Inventory/InventorySideBar/Inventory.js]: done

File: Client/src/Components/Navbar/NavBarButton.js (hunk 7 above)
Comment (berkbozok) [reply Client/src/Components/Navbar/NavBarButton.js:4]: done

File: Client/src/Forms/FormLogin.js (hunk 8 above)
Comment (berkbozok) [reply Client/src/Forms/FormLogin.js]: done

File: Client/src/Forms/FormSuccess.js (hunk 10 above)
Comment (berkbozok) [reply Client/src/Forms/FormSuccess.js]: done

File: Client/src/Forms/useForm.js (hunk 11 above)
Comment (berkbozok) [reply Client/src/Forms/useForm.js]: done

File: Client/src/Forms/validateInfo.js (hunk 12 above)
Comment (berkbozok) [reply Client/src/Forms/validateInfo.js]: done

File: Client/src/Forms/validateLogin.js (hunk 13 above)
Comment (berkbozok) [reply Client/src/Forms/validateLogin.js]: done

File: Client/src/Components/Manufacturing/Manufacturing.js (hunk 6 above)
Comment (Pyxsys) [reply Client/src/Components/Manufacturing/Manufacturing.js:4]: understood. @MichaelDCristiano be sure to add the header in a future commit

File: Client/src/Forms/FormLogin.js (hunk 9 above)
Comment (Pyxsys) [reply Client/src/Forms/FormLogin.js:29]: Addition of Company logo, closes #61

File: Client/src/App.js (hunk 1 above)
Comment (Pyxsys) [reply Client/src/App.js:3]: Confirmation from Michael has been provided.

**Review by Pyxsys — APPROVED** [review]
Requested changes were made. Approved.
Good job @berkbozok!

**Pull request merged**